src/controllers/orderController.js

```diff
@@ -10,4 +12,15 @@ const getOrderList = async (req, res, next) => {
   }
 };
 
-module.exports = { getOrderList };
+const addOrder = async (req, res, next) => {
+  try {
+    const userId = req.decoded;
+    const [totalPrice, ...orderArr] = req.body;
+    await createOrder(userId, orderArr, totalPrice.totalPrice);
+    return res.status(201).json({ message: 'order Created' });
```

Comment:
```suggestion
    const userId = req.decoded;
    const { totalPrice, orderItems } = req.body;
    
    await createOrder(userId, orderItems, totalPrice);
    
    return res.status(201).json({ message: 'order Created' });
```
지금보니 구문이 굉장히 어색하네요. 애초에 전달받고자 하는 데이터 형태를 잘 정돈해놔야 받아서 활용할 때 간단히 활용할 수 있습니다.

Reply:
```JSON
{
    "totalPrice" : 250000,
    "orderItems" : [
        {
            "id"              : 1,
            "productOptionId" : 2,
            "quantity"        : 1
        },
        {
            "id"              : 1,
            "productOptionId" : 2,
            "quantity"        : 1
        }
    ]
}

```